src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(param) {
+      calculator.result += param;
+    },
+
+    subtract(param) {
+      calculator.result -= param;
+    },
+
+    multiply(param) {
+      calculator.result *= param;
+    },
+
+    divide(param) {
+      calculator.result /= param;
```

Comment:
There is a potential division by zero issue here. You should add a check to ensure that `param` is not zero before performing the division to prevent runtime errors.